react-shrine-network-aware-code-splitting/src/index.js

```diff
@@ -48,7 +48,7 @@ ReactDOM.render((
     <ConnectedRouter history={history}>
       <ScrollToTop>
         <MuiThemeProvider theme={theme}>
-          <Router>
+          <Router basename="/react-shrine-network-aware-code-splitting">
```

Comment:
Is this a workaround for our deployment server requiring the full-path to be specified for routing to work at all with the current setup? If so, could we add in a comment above this line to note this?

Reply:
Yes, this is a workaround but typical method for relative URL path.
https://facebook.github.io/create-react-app/docs/deployment
I'll add some comment about this line for sure.

Reply:
set basename prop for Building for Relative Paths from https://facebook.github.io/create-react-app/docs/deployment

I commented like above.